downburst/create.py

```diff
@@ -205,16 +205,18 @@ def create(args):
         rbd_details=rbd_details,
         hypervisor=args.hypervisor
         )
-    dom = conn.defineXML(etree.tostring(domainxml))
+    dom = conn.defineXML(etree.tostring(domainxml).decode())
     dom.create()
     try:
         env = os.environ
         pid = os.getpid()
         # os.getppid() wont return the correct value:
-        ppid = open('/proc/{pid}/stat'.format(pid=pid)).read().split()[3]
-        ppcmdline = open('/proc/{ppid}/cmdline'.format(ppid=ppid)).read().split(b'\x00')
+        stat_path = '/proc/{pid}/stat'.format(pid=pid)
+        ppid = open(stat_path, 'rt').read().split()[3]
```

Comment:
```suggestion
        ppid = open(stat_path).read().split()[3]
```

it's well known that python2 and python3 `open()` file in `rt` by default. no need to be explicit.

Reply:
thanks, fixed